### PR TITLE
a bit of clean-up

### PR DESCRIPTION
**Pull request by cos**

### CODE REVIEW

**Comment by cos** [comment]
Jenkins, test this please.

**Comment by dwightguth** [comment]
lgtm

**Comment by cos** [comment]
take another look, thanks!

File: kernel/src/main/java/org/kframework/kompile/Kompile.java [hunk 1]
```diff
@@ -86,44 +89,63 @@ public Kompile(FileUtil files, KExceptionManager kem) {
...
+        return Definition(withKSeq, d.mainSyntaxModule(), immutable(allModules));
+    }
 
-        Module withKSeq = Module("EXECUTION",
-                Set(concretized, kseqModule),
-                Collections.<Sentence>Set(), Att());
+    class AddProgramModule implements Function<Definition, Definition> {
 
-        final BiFunction<String, Source, K> pp = getProgramParser(parsedDef.getModule(mainProgramsModule).get(), programStartSymbol);
+        private final String mainProgramsModuleName;
```
Comment (dwightguth) [inline kernel/src/main/java/org/kframework/kompile/Kompile.java]: I don't think you need this field, I think you can just access Definition.mainSyntaxModule can't you?

Comment (cos) [inline kernel/src/main/java/org/kframework/kompile/Kompile.java]: yep, right

**Pull request merged**